.github/ISSUE_TEMPLATE/android.md

```diff
@@ -0,0 +1,35 @@
+---
+name: Android
+about: Template for issues about the android code/platform
+title: '[Android]'
+labels: android
+assignees: simonzhexu
+---
+<!-- 
+Only use the GitHub Issues section if you discovered issues with the code itself. Do not mistake the Issues page as a help desk. You can ask for help at [Stack Overflow](https://stackoverflow.com/). 
+-->
+
+**Specifications**  
+OS Version: [FILL IN]
+Device Manufacturer: [FILL IN]
+
+**Stickers**
+- [ ] My stickers do comply with the sticker requirements (512x512 and < 100 KB)
+- [ ] My tray icon does comply with the requirements (96x96 and < 50 KB)
+- [ ] I am using [WebP] images for stickers.
+- [ ] I am using [PNG] images for stickers.
+
+**Problem Description**  
+<!-- FILL IN WITH DETAILED DESCRIPTION OF THE PROBLEM -->
```

Comment:
I would prefer this not being in a comment, perhaps something like [ FILL IN WITH DETAILED DESCRIPTION OF THE PROBLEM ]

This way, people will not mistakenly put their description in the comment. 

Same for Reproduction steps, and links

Also for other types.